static/Makefile

```diff
@@ -17,6 +17,12 @@ export COMPOSE_DIR
 export SCAN_DIR
 
 export STATIC_VERSION
+
+# Select the default version of containerd based on the docker engine source
+# we need this variable here for naming the produced .tar.gz file.
+# TODO containerd binaries should be built as part of containerd-packaging, not as part of docker/docker-ce-packaging
+CONTAINERD_VERSION?=v$(shell grep "ARG CONTAINERD_VERSION" "$(ENGINE_DIR)/Dockerfile.windows" | awk -F'=' '{print $$2}')
```

Comment:
Sorry, this is super ugly, but we needed the containerd version for the `tar.gz` name 😞 🤷‍♂️

Reply:
I think it's fine in the meantime as `Dcokerfile.Windows` is rarely changed, except for bumps. Would have been better to `$(containerd --version)` but we can't atm as we are not sandboxed and would not work for win/darwin bins.

Reply:
Yeah, this is al temporary, so not too worried about it